Profile/Profile/Presentation/Profile/Subviews/ProfileSupportInfo.swift

```diff
@@ -0,0 +1,187 @@
+            viewModel.contactSupport().map(supportInfo)
+            viewModel.config.agreement.tosURL.map(terms)
+            viewModel.config.agreement.privacyPolicyURL.map(privacy)
+            viewModel.config.agreement.cookiePolicyURL.map(cookiePolicy)
+            viewModel.config.agreement.dataSellContentURL.map(dataSellContent)
+            viewModel.config.faq.map(faq)
+            version
+        }
+        .cardStyle(
+            bgColor: Theme.Colors.textInputUnfocusedBackground,
+            strokeColor: .clear
+        )
+    }
+
+    @ViewBuilder
+    private func supportInfo(url: URL) -> some View {
+        button(
+            linkViewModel: .init(
+                url: url,
+                title: ProfileLocalization.contact
+            ),
+            isEmailSupport: true
+        )
+
+    }
+
+    private func terms(url: URL) -> some View {
+        navigationLink(
+            viewModel: .init(
+                url: url,
+                title: ProfileLocalization.terms
+            )
+        )
+    }
+
+    private func privacy(url: URL) -> some View {
+        navigationLink(
+            viewModel: .init(
+                url: url,
+                title: ProfileLocalization.privacy
+            )
+        )
+    }
+
+    private func cookiePolicy(url: URL) -> some View {
+        navigationLink(
+            viewModel: .init(
+                url: url,
+                title: ProfileLocalization.cookiePolicy
+            )
+        )
+    }
+
+    private func dataSellContent(url: URL) -> some View {
+        navigationLink(
+            viewModel: .init(
+                url: url,
+                title: ProfileLocalization.doNotSellInformation
+            )
+        )
+    }
+
+    private func faq(url: URL) -> some View {
+        button(
+            linkViewModel: .init(
+                url: url,
+                title: ProfileLocalization.faq
+            )
+        )
+    }
+
+    @ViewBuilder
+    private func navigationLink(viewModel: LinkViewModel) -> some View {
+        NavigationLink {
+            WebBrowser(
+                url: viewModel.url.absoluteString,
+                pageTitle: viewModel.title
+            )
+        } label: {
+            HStack {
+                Text(viewModel.title)
+                    .multilineTextAlignment(.leading)
+                Spacer()
+                Image(systemName: "chevron.right")
+            }
+        }
+        .foregroundColor(.primary)
+        .accessibilityElement(children: .ignore)
+        .accessibilityLabel(viewModel.title)
+        Rectangle()
+            .frame(height: 1)
+            .foregroundColor(Theme.Colors.textSecondary)
+    }
+
+    @ViewBuilder
+    private func button(linkViewModel: LinkViewModel, isEmailSupport: Bool = false) -> some View {
+        Button {
+            if isEmailSupport { viewModel.trackEmailSupportClicked() }
+            UIApplication.shared.open(linkViewModel.url)
```

Comment:
Before opening the URL can you please check whether the link is openable or not? If not it would be nice if you could show an error/info message.